test/e2e/plugin_lifecycle/plugin_lifecycle_test.go

```diff
@@ -138,7 +138,7 @@ var _ = framework.CLICoreDescribe("[Tests:E2E][Feature:Plugin-lifecycle]", func(
 			// validate above plugin delete with plugin list command, plugin list should return 0 plugins
 			pluginsList, err := framework.GetPluginsList(tf, true)
 			Expect(err).To(BeNil(), "should not get any error for plugin list")
-			Expect(len(pluginsList)).Should(Equal(0), "there should not be any plugins available after uninstall all")
+			Expect(len(pluginsList)).Should(Equal(1), "there should be only one plugin available after uninstall all")
```

Comment:
I assume this is because the `GetPluginList` will reinstall the essential plugins. Can we add that in the comment or message on the right side?

Reply:
makes sense. Added comment